<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifywindow</class>
 <widget class="QMainWindow" name="modifywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入要修改的学生学号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="StuNumlineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="modStulabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>待显示学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QWidget" name="showmodwidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>857</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="7">
      <widget class="QLineEdit" name="classlineEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="stunumlineEdit"/>
     </item>
     <item row="1" column="8">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>绩点</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="sexlineEdit"/>
     </item>
     <item row="1" column="9">
      <widget class="QLineEdit" name="gpalineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>学号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>班级</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="namelineEdit"/>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>请输入该学生修改后的信息</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>确定修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
